d2l-button-behavior.html

```diff
@@ -23,6 +23,25 @@
 			type: Boolean,
 			reflectToAttribute: true
 		},
+		/**
+		 * Indicates whether the element, or another grouping element it
+		 * controls, is currently expanded or collapsed.
+		 */
+		ariaExpanded: {
+			type: String
+		},
```

Comment:
*nit* can be simplified with `ariaExpanded: String`, or are we not using that shortcut?